.github/workflows/render-all.yml

```diff
@@ -1,15 +1,22 @@
-name: Render All Output
+# Candace Savonen 2021
+# Updated May 2024
+
+name: Render all output courses
 
 on:
   workflow_dispatch:
   push:
-    branches: [ main ]
+    branches: [ main, staging ]
```

Comment:
Do we need the staging branch?

Reply:
I like to leave this in because ideally people should use staging branches, so if they choose to do this on a repo its here for that. But if not, it doesn't hurt anything to leave it in.